js/form.js

```diff
@@ -0,0 +1,41 @@
+// Перевод страницы в неактивное состояние
+// 1. Форма заполнения информации об объявлении
+const adForm = document.querySelector('.ad-form');
+const header = adForm.querySelector('.ad-form-header');
+const interactiveFormElements = adForm.querySelectorAll('.ad-form__element');
+// 2. Форма с фильтрами
+const mapFilters = document.querySelector('.map__filters');
+const interactiveMapElements = mapFilters.querySelectorAll('.map__filter');
+const features = mapFilters.querySelector('.map__features');
+
+// Функция перевода страницы в неактивное состояние
+function makePageInactive () {
+  header.setAttribute('disabled', 'disabled');
+  adForm.classList.add('ad-form--disabled');
+  interactiveFormElements.forEach((item) => {
+    item.setAttribute('disabled', 'disabled');
```

Comment:
```suggestion
    item.disabled = true;
```
(подправить для всех)